### ISSUE TITLE
Reorder music, library, artists, and albums tabs

### ISSUE THREAD

**Issue by yssasha** [z-huang/music#1070]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

Library feature will be merged with Liked playlist. That means "Heart button" automatically adds song to the Library. And if it's clicked on the Artist page it means I subscribed to an Artist. So the Add to the library button on the bottom of the play screen will be used as a shortcut for "Adding to a selected playlist". An indicator showing if I liked the song or saved it in the playlist would be cool.
Adding categories to the Playlists tab. It will show all of the Music or Artists or Albums saved in playlists instead of different tabs on the bottom of the screen. It would be cool if there was some indicator showing if I liked/subscribed to an artist or not.
In other words why split Music/Artists/Albums/Playlists in different tabs in the bottom of the app, when we can include all of the above as the categories in the new tab "Saved".

### Why do you want this feature?

It would make app navigation more logical and intuitive.

**Comment by z-huang** [comment]
Yeah, that's your design of how the library should be. But I made this app because I have my own perspective of how a library should be. It's the core of this app and it's not that easy to make a huge change. Anyways, thanks for your idea.
